src/main/java/org/jmhsrobotics/frc2025/subsystems/elevator/Elevator.java

```diff
@@ -25,19 +26,19 @@ public Elevator(ElevatorIO elevatorIO) {
   @Override
   public void periodic() {
     elevatorIO.updateInputs(inputs);
-    stage1.setLength(inputs.positionMeters / 2);
-    carriage.setLength(inputs.positionMeters / 2);
+    stage1.setLength(inputs.heightMeters / 2);
+    carriage.setLength(inputs.heightMeters / 2);
   }
 
-  public boolean atGoal() {
-    return false;
+  public boolean atGoal(double setPoint) {
+    return Math.abs(inputs.heightMeters - setPoint) < Constants.ElevatorConstants.kHeightTolerance;
   }
```

Comment:
Why are we passing the goal to this subsystem. we should just be able to get the setpoint and check. So then you dont need to pass a goal into it.